src/Twig/HashidsExtension.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Roukmoute\HashidsBundle\Twig;
+
+use Roukmoute\HashidsBundle\Hashids;
+
+class HashidsExtension extends \Twig_Extension
+{
+    /**
+     * @var \Hashids\Hashids;
```

Comment:
yes, you're right since `Roukmoute\HashidsBundle\Hashids` already expands `\Hashids\Hashids`